engine/index.js

```diff
@@ -8,20 +8,97 @@ const fixtureDir = path.resolve('./features');
 const fixtureParser = new FixtureParser(fixtureDir);
 const browserParser = new BrowserParser();
 
+function normalizeStatus(status) {
```

Comment:
we might want to add some JSON file later to make this easier to maintain. not in this pull request though.